modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/configuration/AMImageAttributeMapping.java

```diff
@@ -75,6 +76,22 @@ public static AMImageAttributeMapping fromProperties(
 			).build());
 	}
 
+	public static AMImageAttributeMapping fromRawFileVersion(
+		FileVersion fileVersion) {
+
+		return new AMImageAttributeMapping(
+			HashMapBuilder.<AMAttribute<AMImageProcessor, ?>, Optional<?>>put(
+				AMAttribute.getContentLengthAMAttribute(),
+				Optional.of(fileVersion.getSize())
+			).put(
+				AMAttribute.getContentTypeAMAttribute(),
+				Optional.of(fileVersion.getMimeType())
+			).put(
+				AMAttribute.getFileNameAMAttribute(),
+				Optional.of(fileVersion.getFileName())
+			).build());
+	}
+
```

Comment:
Can you rename this method to `fromFileVersion()`?

Reply:
Yes, no problem